locale/translations/zh_CN.json

```diff
@@ -270,6 +270,7 @@
     "form.feed.label.rewrite_rules": "重写规则",
     "form.feed.label.blocklist_rules": "封锁规则",
     "form.feed.label.keeplist_rules": "许可规则",
+    "form.feed.label.urlrewrite_rules": "URL Rewrite Rules",
```

Comment:
You can use Google Translate or similar...

Reply:
Updated all language files with translation from Google Translate.